### PR TITLE
[JENKINS-59037] Allow passing values from casc yaml to job dsl.

### PR DESCRIPTION
**Pull request by rudolfwalter**
An implementation of the improvement proposed in [JENKINS-59037](https://issues.jenkins-ci.org/browse/JENKINS-59037).

### CODE REVIEW

**Comment by daspilker** [comment]
Looks good to me, but pinging @casz for a review.

File: job-dsl-plugin/src/main/groovy/javaposse/jobdsl/plugin/casc/SeedJobConfigurator.java [hunk 1]
```diff
@@ -10,7 +10,10 @@
 import io.jenkins.plugins.casc.impl.attributes.MultivaluedAttribute;
 import io.jenkins.plugins.casc.model.CNode;
 import io.jenkins.plugins.casc.model.Mapping;
-import java.util.Map;
+
+import java.util.*;
```
Comment (casz) [inline job-dsl-plugin/src/main/groovy/javaposse/jobdsl/plugin/casc/SeedJobConfigurator.java]: apart from the star import, this looks good to me!

Comment (rudolfwalter) [reply job-dsl-plugin/src/main/groovy/javaposse/jobdsl/plugin/casc/SeedJobConfigurator.java]: Oops, my IDE did that and I didn't notice :(

Comment (daspilker) [reply job-dsl-plugin/src/main/groovy/javaposse/jobdsl/plugin/casc/SeedJobConfigurator.java]: Yes, please remove the star import and keep the existing order.

Comment (rudolfwalter) [reply job-dsl-plugin/src/main/groovy/javaposse/jobdsl/plugin/casc/SeedJobConfigurator.java]: Done.

File: job-dsl-plugin/src/main/groovy/javaposse/jobdsl/plugin/casc/SeedJobConfigurator.java [hunk 2]
```diff
@@ -23,6 +25,7 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Set;
+import java.util.stream.Stream;
```
Comment (casz) [inline job-dsl-plugin/src/main/groovy/javaposse/jobdsl/plugin/casc/SeedJobConfigurator.java]: The import order still looks funky but I guess that is the job for the maintainer to fix 😅 
In the repos, I maintain I prefer to keep A-Z and static last, no star import at all even in tests.

Comment (daspilker) [reply job-dsl-plugin/src/main/groovy/javaposse/jobdsl/plugin/casc/SeedJobConfigurator.java]: Yes, I am going to fix that.

Comment (casz) [reply job-dsl-plugin/src/main/groovy/javaposse/jobdsl/plugin/casc/SeedJobConfigurator.java]: @daspilker you can have a look at https://github.com/casz/configuration-as-code-plugin we have added IDEA codestyle and checkstyle into maven.

**Pull request merged**